openpype/hosts/max/plugins/publish/validate_no_max_content.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from pymxs import runtime as rt
+
+
+class ValidateMaxContents(pyblish.api.InstancePlugin):
+    """Validates Max contents.
+
+    Check if MaxScene container includes any contents underneath.
+    """
+
+    order = pyblish.api.ValidatorOrder
+    families = ["camera",
+                "maxScene",
+                "maxrender"]
```

Comment:
Wait, `maxrender` doesn't need children right? Or does it?
```suggestion
                "maxScene"]
```

Reply:
you need to have assigned the shot camera as the children of the container before rendering.